src/StaticFAQ/ArticleFeedback/ScreenAdditionalFeedback.js

```diff
@@ -7,21 +7,32 @@ import { Close } from '@kiwicom/orbit-components/lib/icons';
 
 import { Box } from '../../common';
 import createComment from '../../mutations/CreateCommentMutation';
-import screensList from './screensList';
+import screenList from './screenList';
+import commentTypeList from './commentTypeList';
+import { simpleTracker } from '../../helpers/analytics/trackers';
 
 type Props = {|
   changeScreen: (nextScreen: string) => void,
   articleId: string,
+  commentType: string,
+  clearCommentType: () => void,
 |};
+
 type State = {|
   comment: string,
-  error: boolean,
 |};
 
+const commentTypeValues = {
+  [commentTypeList.DONT_LIKE]: 'DONT_LIKE',
```

Comment:
:thinking: maybe a little too complicated :smile:

Reply:
these keys are used in multiple places, so I wanted to decouple them from the components, that's why I have a map object for them in a separate file 🙃